timesheet_helpdesk_ticket/models/helpdesk_ticket_team.py

```diff
@@ -0,0 +1,26 @@
+from odoo import models, fields, api
+
+class HelpdeskTicketTeam(models.Model):
+    _inherit = 'helpdesk.ticket.team'
+
+    archive = fields.Boolean('Archive', related='active', default=True, store=True, readonly=False)    
```

Comment:
This is out-topic

Reply:
Removed for this PR, this change is in https://github.com/OCA/helpdesk/pull/126